### ISSUE TITLE
`view` and `list` commands should have special handling for conflicted definitions

### ISSUE THREAD

**Issue by francisdb** [unisonweb/unison#296]
```
master> add
...
⚠️  I didn't try to update these definitions because the names are conflicted in the current branch
(i.e. already associated with multiple definitions):

  replicate : Nat -> a -> [a]

  Tip: Use `view replicate` to view the conflicting definitions and `rename replicate#abc
       replicate2` to give each definition a distinct name. Alternatively, use `resolve
       replicate#abc` to make `replicate#abc` the canonical `replicate` and remove the name from
       the other definitions.

master> view replicate
replicate : Nat -> a -> [a]
replicate n a = Stream.toSequence (Stream.take 1 (Stream.constant a))

replicate : Nat -> a -> [a]
replicate n a = Stream.toSequence (Stream.take n (Stream.constant a))
```

The shown message mentions `#abc` but this is not visible in `view`?

**Comment by pchiusano** [comment]
@francisdb you are on the bleeding edge :)

`view` and `list` don't yet have any special handling for conflicted definitions, but they should (they should show the first few hash characters for any conflicted symbol). I'm updating this issue to track.

**Comment by pchiusano** [comment]
Expected behavior - when rendering a list of definitions or signatures, if there are multiple terms or types with the same names, the first few hash characters should be appended to the names as they are displayed.

**Comment by aryairani** [comment]
For two conflicted names that refer to a constructor of the same type, it should print and parse with `shortunique hash <> show cid’ and not `shortunique $ show hash <> show cid`.

**Comment by aryairani** [comment]
Fixed in #354.
